pkg/network/node/vnids.go

```diff
@@ -136,6 +136,18 @@ func (vmap *nodeVNIDMap) getVNID(name string) (uint32, error) {
 	return 0, fmt.Errorf("failed to find netid for namespace: %s in vnid map", name)
 }
 
+func (vmap *nodeVNIDMap) findDuplicateNetID(namespace string, netID uint32) bool {
+	names := vmap.GetNamespaces(netID)
+	for _, name := range names {
+		if name != namespace {
```

Comment:
Shouldn't this be `if name == namspace`?

Reply:
Or no, this seems right actually given that we're retrieving based on `netID`

Reply:
no the logic I have is right if we found netID but in different namespace we indicate the duplicate